src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd

```diff
@@ -153,6 +153,11 @@ echo Grant app pool %_wcfServiceName% "Read" access to %_wcfTestDir% and its sub
 call :Run icacls %_wcfTestDir% /grant:r "IIS APPPOOL\%_wcfServiceName%":(OI)(CI)R /Q
 if ERRORLEVEL 1 goto :Failure
 
+:: Unlock the configuration of sslFlags
```

Comment:
NIT: this comment feels a little redundant given the "echo" one line below :smile: